build/evergreen_config_lib/functions.py

```diff
@@ -562,6 +587,20 @@
         fi
         ''', test=False)
     )),
+    ('prepare csfle venv', Function(
+        shell_exec(r'''
```

Comment:
`prepare csfle venv` is always called before `run kms servers`.

Instead of a new function, consider putting the `shell_exec` as the first command of the `Function` in `run kms servers`.